src/appleseed/renderer/meta/tests/test_localsampleaccumulationbuffer.cpp

```diff
@@ -50,7 +50,7 @@ using namespace std;
 
 TEST_SUITE(Renderer_Kernel_Rendering_LocalSampleAccumulationBuffer)
 {
-    bool honors_crop_window(const AABB2u crop_window, const bool undo_premultiplied_alpha)
+    bool honors_crop_window(const AABB2u crop_window)
```

Comment:
Pass `crop_window` by const ref (the problem was already there, but let's just fix it now).